docs/quick-start.md

```diff
@@ -40,7 +40,7 @@ RDDs have _[actions](programming-guide.html#actions)_, which return values, and
 
 {% highlight scala %}
 scala> textFile.count() // Number of items in this RDD
-res0: Long = 126
+res0: Long = 99
```

Comment:
From http://spark.apache.org/docs/latest/quick-start.html
without cache
![screen shot 2016-08-15 at 17 12 33](https://cloud.githubusercontent.com/assets/5894707/17660363/8693aeea-630b-11e6-93b2-09c438e8f77f.png)
and with cache
![screen shot 2016-08-15 at 17 12 44](https://cloud.githubusercontent.com/assets/5894707/17660387/a58635ac-630b-11e6-8541-36003860ed5e.png)